packages/website/src/hooks/useClipboard.ts

```diff
@@ -7,12 +7,13 @@ export type useClipboardResult = [copied: boolean, copy: () => void];
 export function useClipboard(code: () => string): useClipboardResult {
   const [copied, setCopied] = useDebouncedToggle(false);
 
-  const copy = useCallback(() => {
-    // eslint-disable-next-line @typescript-eslint/no-floating-promises
-    void navigator.clipboard.writeText(code()).then(() => {
-      setCopied(true);
-    });
-  }, [setCopied, code]);
+  const copy = useCallback(
+    () =>
+      void navigator.clipboard.writeText(code()).then(() => {
+        setCopied(true);
+      }),
+    [setCopied, code],
+  );
```

Comment:
[Question] Why this change? Seems unrelated & something to be reverted to me?

Reply:
Nope — with ESLint 9, it now reports unused `eslint-disable` comments, like this one 🎉